lm_eval/tasks/leaderboard/ifeval/README.md

```diff
@@ -0,0 +1,45 @@
+# IFEval
```

Comment:
I think we should remove all READMEs for individual tasks and just have one high level README with all info + link to the blog

Reply:
Or if we want one readme per task some readmes are missing + all should contain the info that this is the implementation checked by the leaderboard team.

Reply:
Aggreed, moved everything to one README